StitchMDBKit/StitchMDBKit/StitchMDBKit.h

```diff
@@ -0,0 +1,19 @@
+//
```

Comment:
[q] Is this file necessary because of pod?

Reply:
I believe it's so people can import us from obj-c projects.

Reply:
So is it? This is new code.